tests/fixtures/application_fixtures.py

```diff
@@ -1446,7 +1446,6 @@ def privacy_notice(db: Session) -> Generator:
             "name": "example privacy notice",
             "notice_key": "example_privacy_notice",
             "description": "a sample privacy notice configuration",
-            "origin": "privacy_notice_template_1",
```

Comment:
Notices now have proper "origin" FK's to the template they were created from where applicable so we can't put random strings here any longer.